widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.styles.ts

```diff
@@ -207,8 +207,14 @@ export const Separator = styled('div', {
 });
```

Comment:
It's better if the RightSection has a justify-content: flex-end property. The vertical line has a space with the horizontal line in the loading state.

Also, I'm not sure why `Steps` height is 102px. It seems to have changed in a previous commit, but it doesn't match figma.